database.go

```diff
@@ -19,7 +19,8 @@ import (
 type database struct {
 	name string
 
-	policies map[string]*RetentionPolicy // retention policies by name
+	policies          map[string]*RetentionPolicy // retention policies by name
+	continuousQueries []*ContinuousQuery          // continuous queries by name
```

Comment:
Comment says by name but it's just an array of pointers.  Sorted by name?  Or, should this be a `map[string]*ContinuousQuery`?